script/check-st2-ok

```diff
@@ -24,9 +26,12 @@ retry_message() {
     echo "Don't worry! Occasionally something goes wrong, and you just have"
     echo "to kick the tires a bit. Try re-provisioning by executing the command:"
     echo ""
-    echo "${KICK_COMMAND}"
+    echo "DEBUG=1 ${KICK_COMMAND}"
     echo ""
     echo "If you don't happen to get the 'OK' after that, give us a shout!"
+    echo "Please also make sure to check and forward us the ${LOG_FILE} log file."
```

Comment:
We could also make it easier to gist the log file, but I don't think we should do this automatically without asking the user first.